substrate/primitives/keyring/src/lib.rs

```diff
@@ -15,6 +15,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
I think this should go below the `//!` doc

Reply:
[28ec96c](https://github.com/paritytech/polkadot-sdk/pull/2044/commits/28ec96ca500113efa4072fe5826dad6b70030567)